pages/01.metakube/01.release-notes/default.en.md

```diff
@@ -9,10 +9,16 @@ page-toc:
 ---
 ### January 2020
 
+### 2020-01-29
+
+* Each cluster now runs [node-problem-detector](https://github.com/kubernetes/node-problem-detector) with
```

Comment:
Also explain what this solves. E.g. "This prevents pods, especially from Jobs or CronJobs, being scheduled and started even though the node has no working DNS or network.".